src/patch-global.cjs

```diff
@@ -0,0 +1,6 @@
+if (!globalThis.URLPattern) {
```

Comment:
Can't you generate the .cjs from the .js in the build step?

Reply:
Not if I use the await, but without I should be able to